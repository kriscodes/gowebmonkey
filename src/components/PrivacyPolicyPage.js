import React from 'react';

const PrivacyPolicyPage = () => (
    <div>
        <h2>Privacy Policy</h2>
        <p>Protecting your private information is our priority. This Statement of Privacy applies to
            https://finannsify-a.herokuapp.com/ and KG Holdings LLC and governs data collection and usage.
            For the purpose of this Privacy Policy, unless otherwise noted, all refrences to KG Holdings LLC
            include https://finansify-a.herokuapp.ccom/ and Kris  Finansified. The Kris Finansifies website
            is a Finances site. By  using the Kris Finansifies  website, you consent to the data practices
            described in this Statement.
        </p>
        <h3>
            Collection of your Personal Information
        </h3>
        <p>
            In order to better provide you with products and services offered on our Site, Kris Finansifies
            may collect personally identifiable information, such as your:
        </p>
        <ul>
            E-mail Address
        </ul>
        <p>
            We do not collect any personal information about you unless you voluntarily
            provide it to us. However, you may be required to provide certain personal information
            to us when you elect to use certain products or services available on the Site.
            These may include: (a) registering for an account on our Site;
            (b) entering a sweepstakes or contest sponsored by us or one of our partners;
            (c) signing up for special offers from selected third parties;
            (d) sending us an e-mail message;
            (e) submitting your credit card or other payment information when ordering and purchasing
            products and services on our Site.
            To wit, we will use your information for, but not limited to, communicating with you
            in relation to services and/or products you have requested from us. We also may gather
            additional personal or non-personal information in the future.
        </p>
        <h3>Use of your Personal Information</h3>
        <p>
            Kris Finansifies collects and uses your personal information to operate its website(s) and
            deliver the services you have requested.
        </p>
        <p>
            Kris Finansifies may also use  your personally identifiable information to inform you of other
            products or services available from Kris Fiansifies and its affiliates.
        </p>
        <h3>
            Sharing Information with Third Parties
        </h3>
        <p>
            Kris Finansifies does not sell, rent or lease its customer lists to third parties.
        </p>
        <p>
            Kris Finansifies may share daa with trusted partners to help perform statistical analysis,
            send you email or postal mail, provide customer support, or ararnge for seliveries.
            All such third parties are prohibited from using  your personal information except to provide
            these services to Kris Finansifies, and they are required to maintain the confidentiality of
            your information.
        </p>
        <p>
            Kris Finansifies may disclose your personal information, without notice if required to do so
            by law or in the good faith belief that such action is necessary to:
            (a) conform to the edicts of the law or comply with legal process served on Kris Finansifies or the site;
            (b) protect and defend the rights or property of Kris Finansifies; and/or
            (c) act under exigent circumstances to protect the personal safety of users of Kris Finansifies, or the public.
        </p>
        <h3>
            Automatically Collected Information
        </h3>
        <p>
            Information about your computer hardware and software may be automatically collected by Kris Finansifies.
            This information may include your IP address, browser type, domain names, access times and
            referring website addresses. This information is used for the operation of the service, to maintain
            quality of the service, and ro provide  general statistics regarding use of the Kris Finansifies website.
        </p>
        <h3>
            Security of oyur Personal Information
        </h3>
        <p>
            Kris Finansifies secures your personal information from unauthorized access, use, or disclosure.
            Kris Finansifies uses the following methods for this purpose:
        </p>
        <ul>
            SSL Protocol
        </ul>
        <p>
            When personal information (such as a credit card number) is transmitted to other websites, it is protected
            through the use of encryption, such as the Secure Sockets Layer(SSL) protocol.
        </p>
        <p>
            We strive to take appropriate security measures to protect against unauthorized access to or alteration of
            your personal information. Unfortunately, no data transmission over the Internet or any wireless network
            can be guaranteed to be 100% secure. As a result, while we strive to protect your personal information,
            you acknowledge that:
            (a) there are security and privacy limitations inherent to the Internet which are beyond our control; and
            (b) security, integrity, and privacy of any and all information and data exchanged between you and us
            through this Site cannot be guaranteed.
        </p>
        <h3>
            Children Under Thirteen
        </h3>
        <p>
            Kris Finansifies does not knowingly collect ppersonally identifiable information from children
            under the age of thirteen. If you are under the age of thirteen, you must ask your parent or
            guardian for pemission to use this website.
        </p>
        <h3>
            Changes to this Statement
        </h3>
        <p>
            Kris Finansifies reserves the right to change this Privacy Policy from time to time. We will notify you
            about significant changes in the way we treat personal information by sending a notice to the primary
            email address specified in your account, by placing a prominent notice on our site, and/or by
            updating any privacy information on this page. Your continued use of the Site and/or Services available
            through this Site after such modifications will constitute  your:
            (a) acknowledgement of the modified Privacy Policy; and
            (b) agreement to abide and be bound by that Policy.
        </p>
        <h3>
            Contact Information
        </h3>
        <p>
            Kris Finansifies welcomes your questions  or comments regarding this Statement of Privacy.
            If you believe that Kris Finansifies has not adhered to this Statement, please contact
            Kris Finansifies at:
        </p>
        <p>
            KG Holdings LLC
        </p>
        <p>
            300 Delaware Avenue Suite 210-A
        </p>
        <p>
            Wilmington, Delaware 19801
        </p>
        <p>Email Adrress:</p>
        <p>kristian@kristiancodes.com</p>
        <p>
            Telephone number:
        </p>
        <p>
            (424) 227-2407
        </p>
        <p>
            Effective as of June 15, 2019
        </p>

    </div>
);

export default PrivacyPolicyPage;