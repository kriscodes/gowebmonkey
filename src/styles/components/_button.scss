.button {
  background: $blue;
  width: 200px;
  border: none;
  color: white;
  display: inline-block;
  font-size: $font-size-large;
  font-weight: 300;
  line-height: 1;
  padding: $s-size;
  margin: $s-size;
  text-decoration: none;
}

.button--link {
  background: none;
}

.button--secondary {
  background: #8888;
}
