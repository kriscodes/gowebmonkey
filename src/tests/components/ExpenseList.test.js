import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseList } from '../../components/ExpenseList';
import expenses from '../fixtures/expenses';
import toJson from 'enzyme-to-json';

test('should render ExpenseList with expenses', () => {
   const wrapper = shallow(<ExpenseList expenses={expenses} />);
   expect(toJson(wrapper)).toMatchSnapshot();
});

test('should render ExpenseList with empty message', () => {
    const wrapper = shallow(<ExpenseList expenses={[]} />);
    expect(toJson(wrapper)).toMatchSnapshot();
});