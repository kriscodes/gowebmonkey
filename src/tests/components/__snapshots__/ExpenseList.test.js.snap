// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList with empty message 1`] = `
<div>
  <p>
    No expenses
  </p>
</div>
`;

exports[`should render ExpenseList with expenses 1`] = `
<div>
  <ExpenseListItem
    amount={495}
    createdAt={0}
    description="coffee"
    id="1"
    key="1"
    note="every day"
  />
  <ExpenseListItem
    amount={250000}
    createdAt={-259200000}
    description="June Rent"
    id="2"
    key="2"
    note="monthly"
  />
  <ExpenseListItem
    amount={5000}
    createdAt={345600000}
    description="Credit Card"
    id="3"
    key="3"
    note=""
  />
</div>
`;
